### PR TITLE
Implement UseCase Cards

### LINKED ISSUE #134: Redesign the "UseCase" Card on Android
The use case card in RNTester needs a redesign.

- Use the Material Design Card
- Make title font bigger
- Improve spacing

### PR DESCRIPTION
**Pull request by YashKumarVerma**
## Summary
- Implements the designs [view from here](https://www.figma.com/file/UZ0WPWEuQfr7dwKvirUTAt/Example-Card-Samples?node-id=0%3A1)
- Theme references removed (not completely) to avoid breaking changes with work done on list views.
- Fixes #134

### CODE REVIEW

**Comment by stealthanthrax** [comment]
@YashKumarVerma , can you #909090 or #a4a4a4 for the text colors?

**Comment by YashKumarVerma** [comment]
#909090
WhatsApp Image 2020-07-24 at 9 02 49 AM


#a4a4a4
WhatsApp Image 2020-07-24 at 9 03 50 AM

Disabled Card
WhatsApp Image 2020-07-24 at 9 08 40 AM


@stealthanthrax @anku255 @manyaagarwal @AnshG714  Please check.

File: packages/RNTester/js/components/RNTesterBlock.js [hunk 1]
```diff
@@ -10,108 +10,94 @@
...
-    ) : null;
+import React, {useState} from 'react';
+import {RNTesterThemeContext} from './RNTesterTheme';
+import {StyleSheet, Text, View} from 'react-native';
 
-    return (
+/** functional component for generating example blocks */
+const RNTesterBlock = ({description, isDisabled, title, children}: Props) => {
+  let descComponent = null;
+  isDisabled = [false, true, false, false, false, false, true, false, true][
```
Comment (anku255) [inline packages/RNTester/js/components/RNTesterBlock.js:27]: Can you please explain tp me why do we need to do this?

Comment (YashKumarVerma) [reply packages/RNTester/js/components/RNTesterBlock.js:27]: this shouldn't have been committed :disappointed:

**Comment by anku255** [comment]
@stealthanthrax, @YashKumarVerma  I have made some changes and this is how it looks now - 

Screenshot 2020-07-27 at 12 57 45 PM



Please suggest any changes if required.

**Pull request merged**